### PR TITLE
update_certs.py was silently failing on windows

### LINKED ISSUE #327: ensure python requests is installed
#306 introduced a dependency on the `requests` module in order to have certs installed, but `requests` is only installed by default on linux. It's silently failing during the windows build process. I am not yet sure if this causes a fallback to the default certs on the machine or causes failures, but either way we should ensure the requests module is available.

### PR DESCRIPTION
**Pull request by mattelser**
fixes #327. It needed `requests` and to explicitly write the cert file as `utf-8`

### CODE REVIEW

**Comment by lilleyse** [comment]
Thanks @mattelser.

Could you also update the Windows installation steps in the developer setup README? Both the "Install manually" and "Install with Chocolatey" sections.

File: scripts/update_certs.py [hunk 1]
```diff
@@ -33,7 +33,8 @@ def main():
         print(f"failed to fetch certificates from {CERT_URL}")
         return -1
 
-    with open(CERT_FILE_PATH, "w") as f:
+    # explicit encoding is required for windows
+    with open(CERT_FILE_PATH, "w", encoding='utf-8') as f:
```
Comment (lilleyse) [inline scripts/update_certs.py:37]: Does this fix this error?

```
1> [CMake] Traceback (most recent call last):
1> [CMake]   File "C:\Users\slilley\Code\cesium-omniverse\scripts\update_certs.py", line 43, in <module>
1> [CMake]     sys.exit(main())
1> [CMake]              ^^^^^^
1> [CMake]   File "C:\Users\slilley\Code\cesium-omniverse\scripts\update_certs.py", line 37, in main
1> [CMake]     f.write(req.text)
1> [CMake]   File "C:\Python311\Lib\encodings\cp1252.py", line 19, in encode
1> [CMake]     return codecs.charmap_encode(input,self.errors,encoding_table)[0]
1> [CMake]            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
1> [CMake] UnicodeEncodeError: 'charmap' codec can't encode character '\u0151' in position 37587: character maps to <undefined>
```

Comment (mattelser) [reply scripts/update_certs.py:37]: Yes

**Comment by mattelser** [comment]
> Could you also update the Windows installation steps in the developer setup README? Both the "Install manually" and "Install with Chocolatey" sections.

good call, fixed

**Review by lilleyse — APPROVED** [review]
Confirmed that this is working on windows after doing a `git clean -dxf`

**Pull request merged**